superset/db_engine_specs.py

```diff
@@ -841,6 +841,16 @@ def fetch_result_sets(cls, db, datasource_type, force=False):
         return BaseEngineSpec.fetch_result_sets(
             db, datasource_type, force=force)
 
+    @classmethod
+    def fetch_data(cls, cursor, limit):
+        from TCLIService import ttypes
+        state = cursor.poll()
+        if state.operationState == ttypes.TOperationState.ERROR_STATE:
+            raise Exception('Query error', state.errorMessage)
+        if cls.limit_method == LimitMethod.FETCH_MANY:
```

Comment:
Let's call `super` here instead of repeating the code in the parent class

Reply:
Done.
Sorry for delay. I was on the vacations :)